src/components/ListItem.css

```diff
@@ -12,3 +12,29 @@
 .ListItem-label {
 	margin-left: 0.2em;
 }
+
+.urgency-tag {
+	margin-left: 0.2em;
+	padding: 0.05em;
+	border: 1px solid gray;
+	border-radius: 0.5rem;
+}
+
+.overdue {
+	background-color: #ea5f89;
+}
+
+.soon {
+	background-color: #ffa600;
+}
+.kind-of-soon {
+	background-color: #f7b7a3;
+}
+
+.not-soon {
+	background-color: #c6d68f;
+}
+
+.inactive {
+	background-color: lightgray;
+}
```

Comment:
I love the use of both color and text! This takes care of both design and a11y. 
One small suggestion (and I wouldn't worry about changing anything right now since we'll have time to work on the CSS, a11y features in the next weeks) - and I only know about this because I used to have a color blind student - colors that are similar in hue (like orange and light orange) could potentially be more challenging to distinguish for some users with color vision deficiencies.

Reply:
Thank you for the suggestion Corina! We've changed the orange color to purple for better contrast, also there is a great post if we want to make adjustment further down to the color palette [Coloring for Colorblindness](https://davidmathlogic.com/colorblind/#%23D81B60-%231E88E5-%23FFC107-%23004D40)